.nvmrc

```
@@ -1 +1 @@
-10.22.0
+18.18.0
```

:bulb: let's use the `lts/hydrogen` tag here instead of being so specific

```suggestion
lts/hydrogen
```

---

Good, I will apply the suggestion